AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.swift

```diff
@@ -35,7 +41,10 @@ let package = Package(
         .watchOS(.v6)
     ],
     products: [
-        .library(name: "AWSClientRuntime", targets: ["AWSClientRuntime"])
+        .library(name: "AWSClientRuntime", targets: ["AWSClientRuntime"]),
+        .library(name: "AWSSDKEventStreamsAuth", targets: ["AWSSDKEventStreamsAuth"]),
+        .library(name: "AWSSDKIdentity", targets: ["AWSSDKIdentity"]),
+        .library(name: "AWSSDKHTTPAuth", targets: ["AWSSDKHTTPAuth"]),
```

Comment:
Several new AWS modules are added.

`AWSSDKIdentity` and `AWSSDKHTTPAuth` are likely not complete; enough types were added to them to allow event stream modularization.